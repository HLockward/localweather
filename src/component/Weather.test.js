import React from 'react';
import {shallow} from 'enzyme';
import Weather from './weather';

describe('Weather', () =>{
    let weather = shallow(<Weather />);
    it('should render correctly', () =>{
        expect(weather).toMatchSnapshot();
    });
});