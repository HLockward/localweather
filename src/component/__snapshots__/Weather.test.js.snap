// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Weather should render correctly 1`] = `
<div>
  Weather
</div>
`;
